src/machine/board_circuitplay_bluefruit.go

```diff
@@ -0,0 +1,125 @@
+// +build circuitplay_bluefruit
+
+package machine
+
+const HasLowFrequencyCrystal = true
+
+// Hardware pins
+const (
+	P0_00 Pin = 0
```

Comment:
These pin names are not board specific, so I think they should be moved into a generic nrf52 file (src/machine/board_nrf52832.go).

Reply:
Done